apps/pagetour-sdk/src/playtour/pagetourplay.ts

```diff
@@ -14,7 +16,11 @@ import { Step } from '../models/step'
 import { PageTourTheme } from '../models/pagetourtheme'
 import { DataStore } from '../common/datastore'
 import { TourTypeEnum } from '../models/tourtypeenum'
-import  { querySelectorDeep } from 'query-selector-shadow-dom'
+import { querySelectorDeep } from 'query-selector-shadow-dom'
+import { PageTourOptions } from '../models/pagetouroptions'
+import { isUndefined } from 'util'
```

Comment:
+1

Reply:
removed as util import as it is unecessary